Azure.ResourceManager.Core/ITaggableResource.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
```

Comment:
should add the non LRO versions of add to
AddTag
AddTagAsync

Reply:
Will open another PR for work item: https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5177